### PR TITLE
Re-upload quickstart file

### PR DESCRIPTION
**Pull request by javerbukh**
### Description

@camipacifici noticed that this file was missing and contained valuable information so we are adding it back.

Fixes #<Issue Number>

### Change log entry

- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+---------------------------
+
+``jdaviz`` provides a command-line tool to start the standalone desktop application in a browser. 
+To see the syntax and usage, from a terminal, type::
+
+    jdaviz --help
+
+Typical usage to load a file into a desired configuration::
+
+    jdaviz [imviz|specviz|cubeviz|mosviz] /path/to/data/file
```
Comment (camipacifici) [inline docs/quickstart.rst:43]: Currently in main this does not work. It requires the `--layout` keyword.

Please add a note about what happens when just running `jdaviz` from command line. I just tested that even if the filepicker is not there yet, user can click the configuration without specifying a file and the configuration will open blank. User can then click IMPORT to select the file from a proper filepicker.

Comment (camipacifici) [reply docs/quickstart.rst:43]: I wonder if the `file` tab should be hidden until we have a proper file picker. What do you think?

Comment (javerbukh) [reply docs/quickstart.rst]: Added that as a note, please let me know what you think!

Comment (javerbukh) [reply docs/quickstart.rst]: Which file tab do you mean?

File: docs/quickstart.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+For example, to load a FITS image into Imviz::
+
+    jdaviz --layout=imviz my_image.fits
+
+To learn more about the various ``jdaviz`` application configurations and loading data,
+see the :ref:`imviz`, :ref:`specviz`, :ref:`cubeviz`, or :ref:`mosviz` tools.
+
+.. note::
+
+   The command ``jdaviz`` can be run from the command line but it is a work in progress.
```
[the remark this answers is not in the document]
Comment (javerbukh) [reply docs/quickstart.rst]: Done!

**Review by camipacifici — APPROVED** [review]
Thank you for the quick fix!

**Review by duytnguyendtn — APPROVED** [review]
Thanks for adding the note about the launcher! LGTM!

**Comment by pllim** [comment]
Hmm why did @orifox add it in #1231 and then remove it in https://github.com/spacetelescope/jdaviz/pull/1444 ?

**Pull request merged**